cmd/util/ledger/migrations/accounts.go

```diff
@@ -102,6 +102,12 @@ func AddMissingKeysMigration(payloads []ledger.Payload) ([]ledger.Payload, error
 		return nil, err
 	}
 
+	err = st.Commit()
```

Comment:
Is this a bugfix?

Reply:
Not exactly, just moved the state commit to the function that created the state.